src/rrdp.rs

```diff
@@ -100,23 +108,38 @@ impl NotificationFile {
     ///
     /// Deltas can be processed using the [`ProcessDelta`] trait.
     pub fn deltas(&self) -> &[DeltaInfo] {
```

Comment:
Returning an empty slice whenever `self.deltas` is `Err(..)` feels dangerous. If this is the intention (in order to avoid changes to the API), perhaps stick a warning in the docstring that this method should be combined with `fn delta_status` or something like that?

edit: I see the documentation for `fn parse_limited` contains such a warning, so all good I guess.

Reply:
I will add a note saying that this will be empty if `delta_status` returns an error.